UPGRADING.md

```diff
@@ -4,6 +4,20 @@ Upgrading
 This document summarizes the environmental changes that need to be made when
 upgrading IRIDA that cannot be automated.
 
+22.05 to 22.09
+--------------
+* This upgrade switches the OAuth2 implementation from using spring-security-oauth to spring-security-oauth2-authorization-server and spring-security-oauth2-resource-server. Every administrator will need to update their deployments to configure a java keystore used for encrypting OAuth2 access tokens. This java keystore will need to be present on all servers which allow api access. Otherwise a token generated on one server will not work on the other server.
```

Comment:
Could you mention that encrypting and transferring OAuth2 access tokens is done using JSON Web Tokens and maybe link to some more information (e.g., https://jwt.io/introduction). That way people have more context as to why they need to setup a Java keystore.

Reply:
Added in  84a40d9